lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -990,11 +1050,53 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
 
   defp format_updated_at(updated_at, now) do
     local_updated_at = Util.to_eastern(updated_at)
+    local_now = Util.to_eastern(now)
 
-    if Date.compare(updated_at, now) == :lt do
-      Timex.format!(local_updated_at, "{M}/{D}/{YY}")
-    else
-      Timex.format!(local_updated_at, "{WDfull}, {h12}:{m} {am}")
+    days_since_update =
+      Timex.diff(Timex.to_date(local_now), Timex.to_date(local_updated_at), :days)
```

Comment:
nit: with additions to the Elixir stdlib over the past few years, Timex is usually not needed for datetime functions and I think we should be aiming to remove it, or at least not introduce new code that depends on it. This could be `days_since_update = DateTime.diff(local_now, local_updated_at, :day)`